lib/arclight/traject/ead2_config.rb

```diff
@@ -0,0 +1,188 @@
+  accumulator << Arclight::NormalizedTitle.new(title, dates).to_s
+end
+
+to_field 'normalized_date_ssm' do |_record, accumulator, context|
+  accumulator << Arclight::NormalizedDate.new(
+    context.output_hash['unitdate_inclusive_ssim'],
+    context.output_hash['unitdate_bulk_ssim'],
+    context.output_hash['unitdate_other_ssim']
+  ).to_s
+end
+
+# Each component child document
+# <c> <c01> <c12>
+# rubocop:disable Metrics/BlockLength
+compose 'components', ->(record, accumulator, _context) { accumulator.concat record.xpath('//*[is_component(.)]', NokogiriXpathExtensions.new) } do
+  to_field 'id' do |record, accumulator, context|
+    accumulator << [
+      context.clipboard[:parent].output_hash['id'],
+      record.attribute('id')&.value&.strip&.gsub('.', '-')
+    ].join('')
+  end
+
+  to_field 'ead_ssi' do |_record, accumulator, context|
+    accumulator << context.output_hash['id'].first
+  end
+
+  to_field 'title_ssm', extract_xpath('./xmlns:did/xmlns:unittitle')
+  to_field 'title_teim', extract_xpath('./xmlns:did/xmlns:unittitle')
+
+  to_field 'unitdate_bulk_ssim', extract_xpath('./xmlns:did/xmlns:unitdate[@type="bulk"]')
+  to_field 'unitdate_inclusive_ssim', extract_xpath('./xmlns:did/xmlns:unitdate[@type="inclusive"]')
+  to_field 'unitdate_other_ssim', extract_xpath('./xmlns:did/xmlns:unitdate[not(@type)]')
+
+  to_field 'normalized_title_ssm' do |_record, accumulator, context|
+    dates = Arclight::NormalizedDate.new(
+      context.output_hash['unitdate_inclusive_ssim'],
+      context.output_hash['unitdate_bulk_ssim'],
+      context.output_hash['unitdate_other_ssim']
+    ).to_s
+    title = context.output_hash['title_ssm'].first
+    accumulator << Arclight::NormalizedTitle.new(title, dates).to_s
+  end
+
+  to_field 'normalized_date_ssm' do |_record, accumulator, context|
+    accumulator << Arclight::NormalizedDate.new(
+      context.output_hash['unitdate_inclusive_ssim'],
+      context.output_hash['unitdate_bulk_ssim'],
+      context.output_hash['unitdate_other_ssim']
+    ).to_s
+  end
+
+  to_field 'component_level_isim' do |record, accumulator|
+    accumulator << 1 + record.ancestors.count { |node| node.name == 'c' }
+  end
+
+  # to_field 'parent_ssm'
+  # to_field 'parent_unittitles_ssm'
+  to_field 'unitid_ssm', extract_xpath('./xmlns:did/xmlns:unitid')
+  # to_field 'repository_ssm'
+  # to_field 'repository_sim'
+  to_field 'collection_ssm', extract_xpath('/xmlns:ead/xmlns:archdesc/xmlns:did/xmlns:unittitle')
+  to_field 'collection_sim', extract_xpath('/xmlns:ead/xmlns:archdesc/xmlns:did/xmlns:unittitle')
+
+  to_field 'extent_ssm', extract_xpath('./xmlns:did/xmlns:physdesc/xmlns:extent')
+  to_field 'abstract_ssm', extract_xpath('./xmlns:did/xmlns:abstract')
+  to_field 'scopecontent_ssm', extract_xpath('./xmlns:scopecontent/xmlns:p')
+  # to_field 'creator_ssm'
+  # to_field 'creator_ssim'
+  # to_field 'creators_ssim'
+  # to_field 'creator_sort'
+  # to_field 'collection_creator_ssm'
```

Comment:
Were these `to_field` comments left intentionally?

Reply:
Yes. These are essentially the todos cribbed from here: https://github.com/projectblacklight/arclight/blob/master/app/models/concerns/arclight/solr_document.rb#L14-L120

Additional fields will also need to be added from work completed in #557 

We didn't want to hold other work up by trying to implement all of the fields first.